### PR TITLE
aws-iam: add page

### PR DESCRIPTION
**Pull request by eshack94**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/aws-iam.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# aws iam
+
+> CLI for AWS IAM - web service to securely control access to AWS services and resources.
```
Comment (navarroaxel) [inline pages/common/aws-iam.md:3]:
```suggestion
> CLI for AWS IAM.
> Web service to securely control access to AWS services and resources.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @eshack94, just 1 comment.

File: pages/common/aws-iam.md (hunk 1 above)
Comment (eshack94) [reply pages/common/aws-iam.md]: Change made. Thanks for the feedback.

Comment (navarroaxel) [reply pages/common/aws-iam.md]: But you remove the `> Web service to securely control access to AWS services and resources.` text

Comment (eshack94) [reply pages/common/aws-iam.md]: Oops sorry. I'll fix that when I'm back at my computer.

File: pages/common/aws-iam.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Describe an IAM policy:
+
+`aws iam get-policy --policy-arn arn:aws:iam::aws:policy/{{policy_name}}`
+
+- List access keys:
+
+`aws iam list-access-keys`
+
+- List access keys for a given user:
```
Comment (navarroaxel) [inline pages/common/aws-iam.md:34]:
```suggestion
- List access keys for a specific user:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/aws-iam.md (hunk 2 above)
Comment (eshack94) [reply pages/common/aws-iam.md:34]: Change made.

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
Oh, please the next time don't push force, is harder for us to track the suggestions.

**Comment by eshack94** [comment]
> Oh, please the next time don't push force, is harder for us to track the suggestions.

@navarroaxel Sorry about that. I usually force push to keep the commit history clean. Is it preferred to squash the commits at the end?

**Comment by navarroaxel** [comment]
@eshack94 yep, we squash before merge.

**Comment by eshack94** [comment]
> @eshack94 yep, we squash before merge.

Got it. Thanks for the time and effort in reviewing.

**Review by sbrl — APPROVED** [review]
Thanks, @eshack94!

**Pull request merged**